pages/docs.html

```diff
@@ -1322,6 +1322,52 @@ <h1 class="main__heading p--1" id="modal">
+                            style="width: 700px; height: 983px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You should not use inline styling

Reply:
It is from carbon's iframe. I did not make it. It is automatically there if I copy from carbon.now.sh.